pkg/controllers/monitoring-stack/controller.go

```diff
@@ -42,6 +46,11 @@ import (
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
+const (
+	grafanaDatasourceOwnerName      = "grafana.datasource.owner/name"
```

Comment:
Can we use the following annotations here?
```
monitoring-stack-operator/owner-name
monitoring-stack-operator/owner-namespace
```

Reply:
Done